okhttp-dnsoverhttps/src/main/java/okhttp3/dnsoverhttps/package-info.kt

```diff
@@ -1,2 +1,20 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 /** A DNS over HTTPS implementation for OkHttp. */
+@file:JvmName("PackageInfo")
+
 package okhttp3.dnsoverhttps
```

Comment:
Do we need this kt class?

Reply:
I'd delete.  it technically provides a package comment, or it did.  But it was mainly used for carrying the non null by default annotation we used pre-kotlin.

Reply:
deleted!